src/view/screens/Home.tsx

```diff
@@ -63,9 +95,24 @@ function HomeScreenReady({
     () => pinnedFeedInfos.map(f => f.feedDescriptor),
     [pinnedFeedInfos],
   )
+
+  const currentStarterPack = useCurrentStarterPack()
+  const setCurrentStarterPack = useSetCurrentStarterPack()
+
+  const starterPackInitialFeed = currentStarterPack?.initialFeed
```

Comment:
Can we move this logic to `selected-feed` instead where we're determining the feed to start on?